### ISSUE TITLE
traefik 2.2 kubernetes ingress doc is incorrect for sticky

### ISSUE THREAD

**Issue by jotterson** [containous/traefik#6916]
### Do you want to request a *feature* or report a *bug*?

Bug

### What did you do?

I tried to use the docs here:

https://docs.traefik.io/v2.2/routing/providers/kubernetes-ingress/

### What did you expect to see?

I expected the doc to be correct.

### What did you see instead?

I found the name of the annotation to be incorrect.  the doc page says the annotation is named 

"traefik.ingress.kubernetes.io/service.sticky"

and 

"traefik.ingress.kubernetes.io/service.sticky: "true""

this does not work.  what does work is 

"traefik.ingress.kubernetes.io/service.sticky.cookie": "true"

### Output of `traefik version`: (_What version of Traefik are you using?_)

```
2.2.1
```

### What is your environment & configuration (arguments, toml, provider, platform, ...)?

```
Azure AKS, kubernetes ingress
```

### If applicable, please paste the log output in DEBUG level (`--log.level=DEBUG` switch)

```
N/A
```

**Comment by rtribotte** [comment]
Hi @jotterson ,

Thanks for your interest in Traefik.

I've noticed that the sticky section for an http service, configured using `"traefik.ingress.kubernetes.io/service.sticky: "true""` annotation, is hidden in the dashboard.
When using the annotation `"traefik.ingress.kubernetes.io/service.sticky.cookie": "true"` the section appears.

Is that also your conclusion ? Is that what you mean when you say that it doesn't work ?
Could you provide a reproducible case ?

Thanks

**Comment by jotterson** [comment]
I don't know about the dashboard; I'm not using it.

I know when I tried to use the annotation 

`traefik.ingress.kubernetes.io/service.sticky: "true"`

as seen in the doc here: https://docs.traefik.io/routing/providers/kubernetes-ingress/

that my sessions were not sticky, but when I changed the annotation to 

`traefik.ingress.kubernetes.io/service.sticky.cookie: "true"`

that my sessions were sticky.

Further, if you look at the docs for some of the other providers, for instance rancher, the sticky sessions are enabled by using, again, with sticky.cookie:

` "traefik.http.services.myservice.loadbalancer.sticky.cookie=true"`

which is very similar to the doc for docker, which calls for 

`- "traefik.http.services.myservice.loadbalancer.sticky.cookie=true"`

which is what leads me to believe the documentation for kubernetes ingress is incorrect.

It seems to me that you already reproduced the problem, though in a different way.

thanks, stay safe.

**Comment by jbdoumenjou** [comment]
Hi @jotterson ,

Thanks for your interest in the project.
Could you please give us a reproducible case ?

Thanks

**Comment by jotterson** [comment]
The software works.  The documentation is incorrect.  I'm not sure how to give you a reproducible test case other than to show you some service definitions, one that does not work, and one that works as expected.

This service is not sticky, it round-robins to multiple web servers, despite using the annotation specified in the online doc at https://docs.traefik.io/v2.2/routing/providers/kubernetes-ingress/#on-service

```
apiVersion: v1
kind: Service
metadata:
  name: web
  annotations:
     traefik.ingress.kubernetes.io/service.sticky: "true"
spec:
  ports:
  - name: http
    port: 80
    targetPort: web-80
  selector:
    app: web
```

but set up this way, the session gets stuck to a particular server, and "sticky" works as expected.

```
apiVersion: v1
kind: Service
metadata:
  name: web
  annotations:
     traefik.ingress.kubernetes.io/service.sticky.cookie: "true"
spec:
  ports:
  - name: http
    port: 80
    targetPort: web-80
  selector:
    app: web
```

**Comment by jbdoumenjou** [comment]
I'm sorry, I have not clearly explained why I am asking for a reproducible case.
We have a system in place on the labels which allows us to write what is in the documentation and technically it should work.
Even if it seems obvious, we are asking for a reproducible case, to save us time and understand the context, which in this case means the configuration of Traefik and the ingress.
The more information we have (conf, logs, etc..) the more efficient we can be :)

**Comment by traefiker** [comment]
Closed by #6938.
